src/Cthulhu.jl

```diff
@@ -277,7 +282,18 @@ function _descend(interp::CthulhuInterpreter, mi::MethodInstance; override::Unio
                 callsite = sub_callsites[cid]
             end
 
-            if info isa GeneratedCallInfo || info isa FailedCallInfo
+            if info isa UncachedCallInfo
+                @warn "Cthulhu can't recurse into the uncached callsite"
+                continue
+                # # or, we can forcibly enter into the frame, which the native interpreter threw away
```

Comment:
I suspect we would want to do that. Otherwise it will be hard for the user to investigate further from this point.

Reply:
Yeah, that's also a valid argument. Applied with some other minor fixes: 9624b3b